frontend/src/components/Modal.tsx

```diff
@@ -1,8 +1,37 @@
+import { useState } from "react";
 import styled from "styled-components";
 
-export const Modal = styled.div`
+const ModalBackground = styled.div`
   position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  justify-content: center;
+  align-items: center;
+  display: flex;
   background: rgb(0, 0, 0, 0.7);
-  width: 100%;
-  height: 100%;
+  backdrop-filter: blur(2px);
 `;
+
+const ModalBox = styled.div`
+  background: whitesmoke;
+  padding: 1em;
+  border-radius: 1em;
+`
+
+export type ModalProps = { children: string };
+export const Modal = ({children,}: ModalProps) => {
+  let [showModal, setShowModal] = useState(true);
+
+  const closeModal = () => {
+    setShowModal(false)
+  }
+  const modal : JSX.Element | null = showModal ? 
+                  <ModalBackground>
+                    <ModalBox onClick={closeModal}> You win!</ModalBox> 
+                  </ModalBackground>

```

Comment:
Could add a small x button and then add some listeners to the background to improve user experience. The listeners would listen for escape key and click on the modal to close it 

Something to think about for the next ticket.

Reply:
good idea, especially the escape key.

Reply:
I personally really like when modals let you click anywhere on the screen outside of the modal to close them. Is this default behavior here or something we'd have to implement?

Reply:
> 

will do this

Reply:
done